webcomponents/tool/src/main/frontend/js/sakai-permissions.js

```diff
@@ -172,6 +172,31 @@ class SakaiPermissions extends SakaiElement {
   }
 
   resetPermissions() {
+
+    this.updateComplete.then(() => {
+
+      let inputs = this.renderRoot.querySelectorAll("input[type='checkbox']");
+      inputs.forEach(elem => {
+
+          let role = elem.getAttribute('data-role');
+          let perm = elem.getAttribute('data-perm');
+          if (role && perm) {
+
+            elem.checked = this.on[role].includes(perm);
+            if ("createEvent" in document) {
+
+                let evt = document.createEvent("HTMLEvents");
+                evt.initEvent("change", false, true);
```

Comment:
Can't you just use "new Event("change", { bubbles: true })" for these lines? Maybe you don't even need to bubble it.

https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events

Reply:
Yes. You are right. I'm still attached to the old ways...